tests.py

```diff
@@ -67,6 +67,53 @@ def tearDownClass(self):
                 logging.warning("{} not removed prior to tests".format(self.outdir))
 
 
+class MCMCSearchTest(Test):
```

Comment:
@dbkeitel opinions?

Reply:
Yes, good idea! The whole test suite could be cleaned up with more inheritance, but the again, it seems it should be migrated from an unholy unittest-pytest mix to pure pytest later, so it's not worth much more effort at this point I'd say.

Could you move this to further down though, just before the first child class (`class MCMCSearch(MCMCSearchTest)`)?

Reply:
Done